src/hl/container.rs

```diff
@@ -26,12 +26,14 @@ impl<'a> Reader<'a> {
     }
 
     /// Set maximum allowed conversion level.
+    #[must_use]
```

Comment:
Maybe remove the new `#[must_use]` instances in this crate as well?

Reply:
`#[must_use]` has been removed from everywhere I've added them